README.md

```diff
@@ -188,23 +189,26 @@ directory. The following commands will do this for you:
 
 ```shell
 $ mkdir -p ~/balenaProduction
+$ cd ~/balenaProduction
+$ echo "balenaUrl: 'balena-cloud.com'" > balenarc.yml
+$ echo "dataDirectory: '.'" >> balenarc.yml
+
 $ mkdir -p ~/balenaStaging
-$ echo "balenaUrl: 'balena-staging.com'" > ~/balenaStaging/balenarc.yml
-$ echo "balenaUrl: 'balena-cloud.com'" > ~/balenaProduction/balenarc.yml
+$ cd ~/balenaStaging
+$ echo "balenaUrl: 'balena-staging.com'" > balenarc.yml
+$ echo "dataDirectory: '.'" >> balenarc.yml
 ```
 
 Note that we do *not* prefix the `balenarc.yml` file with a `.` to avoid hiding
-it.
+it. Note also that some lines above use `'>'` and others use `'>>'` as the shell
```

Comment:
```suggestion
it. Note also that some lines above use `>` and others use `>>` as the shell
```

Reply:
My original version (pre review) didn't have the quotes, :-) then I thought it didn't read nicely, so I added the quotes. This is how it looks without the quotes:

> Note also that some lines above use `>` and others use `>>` as the shell

_vs._

> Note also that some lines above use `'>'` and others use `'>>'` as the shell

Perhaps it depends on the background contrast on the user's screen. I like it better with the quotes. :-) There is no danger of the user copying and pasting the quotes because the actual command line examples do not have quotes.

Thanks for the review! 👍